### PR TITLE
add this file

### PR DESCRIPTION
**Pull request by MariaKhachatryan**

### CODE REVIEW

**Review by annasargsyan91 — APPROVED** [review]

**Review by GayaneMg — APPROVED** [review]

File: test.txt [hunk 1]
```diff
@@ -4,3 +4,4 @@ lihdfvbijadn w\d]'
 v sdfdf
 v 
 new comment by artbar85
```
Comment (astxik27) [inline test.txt:6]: astxik_github_commit

**Review by jemma7 — APPROVED** [review]

**Review by jemma7 — APPROVED** [review]

**Review by Marine-Saryan — APPROVED** [review]

**Review by HaykanushGrigoryan — APPROVED** [review]

**Review by astxik27 — COMMENTED** [review]
Astx review

**Comment by astxik27** [comment]
Ok

File: test.txt (hunk 1 above)
Comment (astxik27) [inline test.txt:6]: Ok

**Review by astxik27 — APPROVED** [review]
Ok

**Review by Anzhelika-g — APPROVED** [review]

**Pull request merged**